### PR TITLE
Dawnpm menu tweaks

### PR DESCRIPTION
**Pull request by dawnpm**

### CODE REVIEW

**Comment by dawnpm** [comment]
@amanifarooque please take a look at the sidenav changes I made, particularly to the about-us side nav.

File: _config.yml [hunk 1]
```diff
@@ -151,24 +155,29 @@ primary_navigation:
...
+      url: "/about/policy/"
+    - name: For Developers
+      url: "/developer/"
+    - name: Security and Compliance
+      url: "/about/security.html"
+    - name: Share Your Feedback
+      url: "/about/feedback.html"
+  - name: Related Resources
+    children: 
+    - name: For Developers
```
Comment (dawnpm) [inline _config.yml:179]: @amanifarooque can you figure out why this isn't indenting properly after the site builds?

Comment (amanifarooque) [reply _config.yml:179]: I just npm installed and now local dev is the same (incorrect) indentation. Seems like the USWDS CSS changed a bit - I just committed a change to force the old styling to keep the indentation for now.

**Review by amanifarooque — APPROVED** [review]

**Comment by amanifarooque** [comment]
Looks good, I'm merging!

**Pull request merged**